zulipterminal/helper.py

```diff
@@ -47,6 +47,9 @@
 )
 
 
+RESOLVED_TOPIC_PREFIX = "✔ "
+
+
```

Comment:
Adding new lines just increases the no. of changes. So, try minimizing your code changes by sticking to only code additions and omissions, not the spacing.